js/components/tab.js

```diff
@@ -172,15 +174,15 @@ class Tab extends ImmutableComponent {
     }
 
     let playIcon = null
+    let iconClass = null
+
     if (this.props.tab.get('audioPlaybackActive') || this.props.tab.get('audioMuted')) {
-      playIcon = <span className={cx({
-        audioPlaybackActive: true,
-        fa: true,
-        'fa-volume-up': this.props.tab.get('audioPlaybackActive') &&
-          !this.props.tab.get('audioMuted'),
-        'fa-volume-off': this.props.tab.get('audioPlaybackActive') &&
-          this.props.tab.get('audioMuted')
-      })}
+      if (this.props.tab.get('audioPlaybackActive') && !this.props.tab.get('audioMuted')) {
+        iconClass = 'fa fa-volume-up'
```

Comment:
These class names (along with the other FontAwesome class names) might fit into the global styles? For example:
```
const globalStyles = {
  classAudioPlayingIcon: 'fa fa-volume-up',
  classAudioMutedIcon: 'fa fa-volume-off',
  //...
}
```